.gitlab-ci.yml

```diff
@@ -39,18 +41,20 @@ build_kinetic:
     ROS_DISTRO: kinetic
   <<: *build_common
   script:
+  # colcon as a Debian package is only available in Kinetic and up
+    - apt-get install -y python3-colcon-common-extensions
   # Install lcov from source as binary installation errors when appending files
     - git clone https://github.com/linux-test-project/lcov.git
     - cd lcov
     - git checkout v1.13
     - make install 
     - cd ..
     - rm -r lcov
-    - catkin_make clean
-    - catkin_make -j2 --cmake-args -DCMAKE_CXX_FLAGS="${CMAKE_CXX_FLAGS} -fprofile-arcs -ftest-coverage" -DCMAKE_C_FLAGS="${CMAKE_C_FLAGS} -fprofile-arcs -ftest-coverage" -DCMAKE_BUILD_TYPE=Debug
+    - colcon build --cmake-args -DCMAKE_CXX_FLAGS="${CMAKE_CXX_FLAGS} -fprofile-arcs -ftest-coverage" -DCMAKE_C_FLAGS="${CMAKE_C_FLAGS} -fprofile-arcs -ftest-coverage" -DCMAKE_BUILD_TYPE=Debug
+    - colcon build --cmake-target tests --cmake-args -DCMAKE_CXX_FLAGS="${CMAKE_CXX_FLAGS} -fprofile-arcs -ftest-coverage" -DCMAKE_C_FLAGS="${CMAKE_C_FLAGS} -fprofile-arcs -ftest-coverage" -DCMAKE_BUILD_TYPE=Debug
```

Comment:
@esteve Please add comment explaining these 2 lines

Reply:
Done in https://github.com/CPFL/Autoware/pull/1704/commits/f018965126d482722d3f5c1fcc6ab0f7f935a7d6